src/main/java/frc/robot/subsystems/endEffector/EndEffectorConstants.java

```diff
@@ -5,5 +5,8 @@ public class EndEffectorConstants {
   public static final int deviceID = 16;
   public static final int GEAR_RATIO = 3;
   public static final double HOLD_CORAL_CURRENT = 10.0;
+  public static final int CORAL_DETECTOR_ID = 2; // Assume 0 for now
+  public static final double EXP_DECAY = 0.02; // exponential decay rate for EMA
+  public static final double SDEV_Decay = 0.02;
```

Comment:
```suggestion
  public static final double SDEV_DECAY = 0.02;
```

Let's go ahead and make it all caps, just to follow convention. And we'll need to update everything according that uses this constant